File: csharp/Algorithms/GCD.cs
namespace Algorithms
{
    public class GCD
    {
        public static int Get(int a, int b)
        {
            if (a == 0)
            {
                return b;
            }
            if (b == 0)
            {
                return a;
            }

            return Get(b, a % b);
        }
    }
}

File: java/src/main/java/grokking/algorithms/GCD.java
package grokking.algorithms;

public class GCD {
    public static int get(int a, int b) {
        if (a == 0) {
            return b;
        }
        if (b == 0) {
            return a;
        }

        return get(b, a % b);
    }
}